qa/evil-tests/src/test/java/org/elasticsearch/bootstrap/EvilJNANativesTests.java

```diff
@@ -80,4 +80,29 @@ public void testSetMaxSizeVirtualMemory() throws IOException {
         }
     }
 
+    public void testSetMaxFileSize() throws IOException {
+        if (Constants.LINUX) {
+            final List<String> lines = Files.readAllLines(PathUtils.get("/proc/self/limits"));
+            if (!lines.isEmpty()) {
```

Comment:
Inside this block we make assertions and return from the method. Otherwise, there is a fail below this block that fails the test because we can not assert anything if for some reason we did not read any content from `/proc/self/limits`